functions/New-DbaAgentOperator.ps1

```diff
@@ -111,6 +117,8 @@ function New-DbaAgentOperator {
         [switch]$IsFailsafeOperator = $false,
         [string]$FailsafeNotificationMethod = "NotifyEmail",
         [switch]$Force = $false,
+        [parameter(ValueFromPipeline)]
+        [Microsoft.SqlServer.Management.Smo.Server[]]$ServerObject,
```

Comment:
Just got one change, please modify `ServerObject` to the standard `InputObject` instead.

Reply:
Thank you for noticing that. I've uploaded those changes.